metacat-common-server/src/main/java/com/netflix/metacat/common/server/usermetadata/UserMetadataService.java

```diff
@@ -45,46 +47,52 @@
      *
      * @param uris list of uris.
      */
-    void deleteDataMetadatas(@Nonnull List<String> uris);
+    default void deleteDataMetadatas(@Nonnull List<String> uris) {
```

Comment:
We shouldn't have to put `@Nonnull` on the method parameters if you've put the annotation on the `package-info.java` file as we all agreed.